x/ccv/provider/keeper/distribution.go

```diff
@@ -238,7 +238,7 @@ func (k Keeper) GetConsumerRewardsPool(ctx sdk.Context) sdk.Coins {
 // ComputeConsumerTotalVotingPower returns the validator set total voting power
 // for the given consumer chain
 func (k Keeper) ComputeConsumerTotalVotingPower(ctx sdk.Context, chainID string) (totalPower int64) {
-	// sum the opted-in validators set voting powers
+	// sum the consumer validators set voting powers
```

Comment:
Changed due to the "The comment in the function ComputeConsumerTotalVotingPower incorrectly states that it sums the voting powers of opted-in validators. Instead, it sums the voting powers of the actual validator set." comment.